.github/workflows/foundry.yml

```
@@ -3,9 +3,23 @@ name: Foundry
 on:
   workflow_dispatch:
   pull_request:
+    paths:
+      - '.github/**'
+      - 'bin/**'
+      - 'src/**'
+      - 'lib/**'
+      - 'certora/**'
+      - 'foundry.toml'
```

Would suggest filtering by file type, rather than directory. We should always rerun ci if solidity is modified:

```suggestion
      - '.github/**'
      - './**/**.sol'
```